mailpoet/lib/WP/Functions.php

```diff
@@ -140,6 +140,10 @@ public function deleteCommentMeta($commentId, $metaKey, $metaValue = '') {
     return delete_comment_meta($commentId, $metaKey, $metaValue);
   }
 
+  public function addOption($option_name, $option_value) {
+    return add_option($option_name, $option_value);
```

Comment:
Yeah, I should have added the full signature and I would if we had decided to keep addOption() as more often than not it makes sense to add an option with autoload set to `no` to avoid performance problems due to bloated options that are unnecessarily autoloaded. Thanks for mentioning it.